webapp/sources/rudder/rudder-core/src/main/resources/hooks.d/policy-generation-node-ready/hook-template

```diff
@@ -1,8 +1,12 @@
 #!/bin/sh
 
-# Hooks parameter are passed by environment variable: 
+# Use 'HOOK_WARN_TIMEOUT' or 'HOOK_KILL_TIMEOUT' to specify a time after which a warning is emitted or the hook is interrupted.
+# For example, 'HOOK_WARN_TIMEOUT = 1 second' (whithout quote, without text on the line after duration) will emit a warning if
```

Comment:
I feel it would be clearer to state
# For example, adding a line ending by 'HOOK_WARN_TIMEOUT = 1 second' (without quote) and followed by an empty line , without text on the following line) will configure the hook to emit a warning if it takes more than 1 second